drivers/spi/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ zephyr_library()
 
 zephyr_library_sources_ifdef(CONFIG_SPI_DW		spi_dw.c)
 zephyr_library_sources_ifdef(CONFIG_SPI_INTEL		spi_intel.c)
-zephyr_library_sources_ifdef(CONFIG_SPI_STM32		spi_ll_stm32.c)
+zephyr_library_sources_jinjagen_ifdef(CONFIG_SPI_STM32	spi_ll_stm32_device.h.jinja2)
```

Comment:
I guess one line is missing here.
But, as commented above, I'll be glad with :-):
```
zephyr_library_sources_jinjagen_ifdef(CONFIG_SPI_STM32	spi_ll_stm32.c)
```